src/index.js

```diff
@@ -0,0 +1,43 @@
+// import _ from 'lodash';
+import './style.css';
+
+const tasks = [
+  {
+    discription: 'Write a  ',
+    complated: false,
+    index: 1,
+  },
+  {
+    discription: 'On page  tasks ',
+    completed: false,
+    index: 2,
+  },
+  {
+    discription: 'The list should aalues for each task.',
+    completed: false,
+    index: 3,
+  },
+  {
+    discription: 'Create a styler t List',
+    completed: false,
+    index: 4,
+  },
+];
+
+const taskList = document.querySelector('.taskList');
+// eslint-disable-next-line no-unused-vars
```

Comment:
- Please remove all disabled linters from your code base, remember linters help to ensure that we write code that is clean and easier to maintain. Please fix this 😄